### ISSUE TITLE
yaml.CSafeDumper not defined

### ISSUE THREAD

**Issue by JukkaL** [python/typeshed#2665]
Both of these imports from yaml work at runtime, but the second one is not defined in the stubs:

```
from yaml import CSafeLoader  # OK
from yaml import CSafeDumper  # Error
```

I saw `CSafeDumper` being used in some internal code.

**Comment by JelleZijlstra** [comment]
Fixed by #2613.

**Comment by Hanaasagi** [comment]
Maybe, I can work on this.
